app/models/journey.rb

```diff
@@ -45,6 +45,10 @@ class Journey < ApplicationRecord
 
   scope :default_order, -> { order(client_timestamp: :asc) }
 
+  scope :updated_at_range, lambda { |from, to|
```

Comment:
We should add a spec for this https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/spec/models/move_spec.rb#L561-L577